cafebabel/static/js/profile-contribution-switch.js

```diff
@@ -0,0 +1,47 @@
+const translationSwitcher = document.querySelector('.translation-switcher'),
+  articleSwitcher = document.querySelector('.article-switcher'),
+  articleList = document.querySelector('.article-list'),
+  translationList = document.querySelector('.translation-list')
+
+translationSwitcher.addEventListener('click', () => {
+  translationSwitcher.classList.add('active')
+  articleSwitcher.classList.remove('active')
+  /*articleList.style.opacity = 0*/
+  fadeOut(articleList)
+  fadeIn(translationList)
+})
+articleSwitcher.addEventListener('click', () => {
+  translationSwitcher.classList.remove('active')
+  articleSwitcher.classList.add('active')
+  /*translationList.style.opacity = 0*/
+  fadeOut(translationList)
+  fadeIn(articleList)
+})
+
+/* http://youmightnotneedjquery.com/ */
+function fadeIn(element) {
+  element.style.opacity = 0
+  var last = +new Date()
```

Comment:
Use `const`.

Reply:
If I use `const` I get `Uncaught TypeError: Assignment to constant variable.` 😢

Reply:
Alright, I missed the following redeclaration, use `let` in this case.

Reply:
ok